lib/server/monalisa-synchronization/MonAlisaSynchronizer.js

```diff
@@ -71,13 +71,22 @@ class MonAlisaSynchronizer {
                 }
             };
 
+            const errors = {};
             for (const monalisaDataPass of dataPasses) {
                 await dataSource.transaction(() => transactionablePipeline(monalisaDataPass))
```

Comment:
Could you rename this `transactionablePipeline` by something that inform the user about what it does?
Please think thoughtfully the name you'll use.

Reply:
applied